src/read/import.rs

```diff
@@ -0,0 +1,208 @@
+//! Support for reading short import files.
```

Comment:
I think this file should be renamed to make it unambiguous that it is about import files as used by MSVC and not import libraries in general.

Reply:
My original thinking was that I didn't want the module name to be too specific because this module could perhaps be expanded to provide ways to help read/aggregate the `.idata$` used in long form imports.

But a more specific name does sound reasonable. My only quibble is that it's not really any more MSVC specific than anything else in the spec. It's well documented and tools other than MSVC use it (notably lld and llvm-dlltool). So I'd rather the name be vendor-agnostic, especially if GNU does eventually add support for it.

Reply:
It is at the very least Windows or COFF specific.

Reply:
Ah true. I did find it hard to fit into existing categories. It's specific to Windows archive files. Maybe I should merge it in to `read::archive` and tweak the struct names.

Reply:
After some more thought I moved it into the `coff` module, though I may still want to fiddle with the names of things.

I think `archive` is more relevant in some ways (this doesn't make sense outside of an archive) but on balance pairing it with `coff` does fit the way things are currently organised. Even if you're skipping over any COFF files in the archive, it's still an MS COFF archive format you're reading. Though tbh I'm still not sure.

Reply:
I think the `read::coff` module makes sense. It is contained in archives, as an alternative to COFF files, rather than being part of the archive format itself. This means the features will work correctly too, since it depends on the `pe` module, which requires either the `coff` or `pe` features.